decode/render.ts

```diff
@@ -277,7 +295,10 @@ export function change(data: InputChangeData, iframe: HTMLIFrameElement): void {
 export function selection(data: SelectionData, iframe: HTMLIFrameElement): void {
     let doc = iframe.contentDocument;
     let s = doc.getSelection();
-    s.setBaseAndExtent(element(data.start as number), data.startOffset, element(data.end as number), data.endOffset);
+    // Wrapping selection code inside a try / catch to avoid throwing errors when dealing with elements inside the shadow DOM.
```

Comment:
>selection [](start = 16, length = 9)

Nice comment. Could you expand what it means to have a selection in the shadow DOM? In particular can a selection include only a part of the shadow DOM or is always going to include  the entire shadow DOM (i.e. the element that the shadow DOM represents.

Reply:
Added the comment in main instrumentation code per our offline discussion.